pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -303,6 +303,8 @@ let
       # Intel GVT-g graphics virtualization supports 64-bit only
       DRM_I915_GVT = whenAtLeast "4.16" yes;
       DRM_I915_GVT_KVMGT = whenAtLeast "4.16" module;
+      # Enable Hyper-V Synthetic DRM Driver
+      DRM_HYPERV = whenAtLeast "5.14" yes;
```

Comment:
Why not as a module like [arch](https://github.com/archlinux/svntogit-packages/blob/packages/linux/trunk/config#L6644) does?

Reply:
That's originally what I had and then I switched it before I sent. I think a module is probably appropriate, though I was somewhat following some other video drivers in here that are set to "yes".

I switched it back to being a module, that's probably appropriate. Someone could add this to the hyperv-image modules.